CMakeLists.txt

```diff
@@ -28,8 +28,15 @@ include(CTest)
 
 include(CMakeDependentOption)
 option(CPPUTEST_STD_C_LIB_DISABLED "Disable the standard C library")
+
+include(CheckCXXSourceCompiles)
+check_cxx_source_compiles(
+  "int main(int argc, char ** argv) { throw 20; }"
+  CPPUTEST_HAVE_EXCEPTIONS
+)
+
 cmake_dependent_option(CPPUTEST_STD_CPP_LIB_DISABLED "Use the standard C++ library"
-  OFF "NOT CPPUTEST_STD_C_LIB_DISABLED" ON)
+  OFF "NOT CPPUTEST_STD_C_LIB_DISABLED;CPPUTEST_HAVE_EXCEPTIONS" ON)
```

Comment:
Is this one accidentally still here ?

Reply:
I'm not sure I understand your question. This line automatically disables the C++ standard library if:
- the C standard library is disabled, or
- exceptions are not supported.

Reply:
Sorry, what I mean is that here is now the only place there still in CPPUTEST_HAVE_EXCEPTIONS. I wondered if that is still needed?

But... now I see I was wrong. Above this, the CPPUTEST_HAVE_EXCEPTIONS is still checked also. So, no more comment :)